### PR TITLE
Break attachments into smaller lines

### PR DESCRIPTION
**Pull request by ronisbr**
Some email clients (like Spark Mail) has problems with emails sent by SMTPClient.jl that has large attachments. After comparing with emails sent with Python, which does not have problems, I identified the root cause as how SMTPClient.jl adds the encoded stream of attachments to the e-mail body.

The encoded string obtained is not wrapper, meaning that we will have one **very** big line for big files. This commit breaks the payload into lines with 75 characters, as Python does. I tested with inline images and some attachments files and everything seems to be working perfectly.

### CODE REVIEW

File: src/user.jl [hunk 1]
```diff
@@ -20,15 +20,32 @@ function encode_attachment(filename::String)
...
 
-    encoded_str = 
+    # Some email clients, like Spark Mail, have problems when the attachment
+    # encoded string is very long. This code breaks the payload into lines with
+    # 75 characters, avoiding those problems.
+    raw_attachment = String(take!(io))
+    buf = IOBuffer()
+    char_count = 0
+
+    for c in raw_attachment
```
Comment (aviks) [inline src/user.jl:30]: can't this loop be written on the buffer (here `io` ) . We have a buffer which we convert to a string, then write back to a buffer and then convert to string once again. Which seems a bit redundant.

Comment (ronisbr) [reply src/user.jl:30]: Hi @aviks ! Probably not. The first buffer encodes the file. Hence, every byte written to it is automatically encoded, meaning we cannot add a break-line to the encoded stream. This functionality should be implemented in `Base64` module of Julia. At the second stage, to break the long lines, we could use just a string and concatenate every character using `*`. However, a buffer for this case is much faster.

Comment (ronisbr) [reply src/user.jl:30]: Hi @aviks !

Can we merge as it is or do you have a new proposal?

**Comment by ronisbr** [comment]
Bump!

File: src/user.jl (hunk 1 above)
Comment (aviks) [reply src/user.jl:30]: I had some thoughts, but clearly I haven't had the time to experiment with that 😄

**Pull request merged**